app/views/job_runs/discovery_report_summary.html.erb

```diff
@@ -1,76 +1,74 @@
-<div class="container">
-  <div class="row mt-5">
-    <div class="col-sm-12">
-      <h1><%= link_to @job_run.batch_context.project_name, @job_run.batch_context %> by <%= @job_run.batch_context.user.email %></h1>
-      <p><%= @job_run.job_type.humanize %> #<%= @job_run.id %></p>
-    </div>
+<div class="row mt-5">
```

Comment:
`.erb` files are showing many lines changed because of indentation changes alone. The substantive change is moving the `"container"` class to a `<main>` tag in `application.html.erb`.